### PR TITLE
En_Prz

### PR DESCRIPTION
**Pull request by Kelebek1**
Before opening this PR, ensure the following:
- `./format.sh` was run to apply standard formatting.
- `make` successfully builds a matching ROM.
- No new compiler warnings were introduced during the build process.
    - Can be verified locally by running `tools/warnings_count/check_new_warnings.sh`
- New variables & functions should follow standard naming conventions.
- Comments and variables have correct spelling.
---

### CODE REVIEW

File: src/overlays/actors/ovl_En_Prz/z_en_prz.c [hunk 1]
```diff
@@ -71,45 +93,442 @@ const ActorInit En_Prz_InitVars = {
...
+void EnPrz_Init(Actor* thisx, GlobalContext* globalCtx) {
+    EnPrz* this = THIS;
+
+    this->unk_20C = 0.01f;
+    this->unk_1E4 = this->actor.world.rot.y;
+
+    Actor_SetScale(&this->actor, 0.01f);
+
+    this->actor.targetMode = 3;
+    this->unk_1E8 = 0xFF;
```
Comment (jpburnett) [inline src/overlays/actors/ovl_En_Prz/z_en_prz.c:111]:
```suggestion
    this->unk_1E8 = 255;
```
Used as a color so decimal can be used

File: src/overlays/actors/ovl_En_Prz/z_en_prz.c [hunk 2]
```diff
@@ -71,45 +93,442 @@ const ActorInit En_Prz_InitVars = {
...
+    this->skelAnime.playSpeed = 2.0f;
+    this->actionFunc = func_80A767A8;
+}
+
+typedef struct {
+    Actor actor;
+    char unk_144[0x184];
+    f32 unk_2C8;
+} EnPrzUnkParent;
+
```
Comment (jpburnett) [inline src/overlays/actors/ovl_En_Prz/z_en_prz.c:316]: Could this just be using EnPr2? The structs seem relatively similar:
```
typedef struct EnPr2 {
    /* 0x0000 */ Actor actor;
    /* 0x0144 */ char unk_144[0x80];
    /* 0x01C4 */ EnPr2ActionFunc actionFunc;
    /* 0x01C8 */ char unk_1C8[0x100];
} EnPr2; // size = 0x2C8
```

**Review by jpburnett — COMMENTED** [review]
Great job, just a comment on a 2 parts.

File: src/overlays/actors/ovl_En_Prz/z_en_prz.c (hunk 2 above)
Comment (hensldm) [reply src/overlays/actors/ovl_En_Prz/z_en_prz.c:316]: Looking at KZ, the parent actor to EnPrz seems to be `EnPr`.

Comment (Kelebek1) [reply src/overlays/actors/ovl_En_Prz/z_en_prz.c:316]: Thanks.

**Review by AngheloAlf — APPROVED** [review]

**Review by hensldm — APPROVED** [review]

**Review by Kenix3 — APPROVED** [review]

**Pull request merged**